src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -648,6 +648,77 @@
     .power 10
     $.equal-to 2655.992279142402
 
+[] > sqrt-check-zero-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          0.0
+          sqrt.
+            number 0.0
+    $.less-than 0.00000000001
+
+[] > sqrt-check-negative-input
+  assert-that > @
+    is-nan.
+      sqrt.
+        number -0.1
+    $.equal-to TRUE
+
+[] > sqrt-check-float-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          2.0
+          sqrt.
+            number
+              4.0
+    $.less-than 0.00000000001
+
+[] > sqrt-check-int-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          9.0
+          sqrt.
+            number
+              81
+    $.less-than 0.00000000001
+
+[] > sqrt-check-nan-input
+  assert-that > @
+    is-nan.
+      sqrt.
+        number nan
+    $.equal-to TRUE
+
+[] > sqrt-check-max-int
+  9223372036854775807 > max-int!
+  assert-that > @
+    abs.
+      number
+        minus.
+          3037000499.97605
+          sqrt.
+            number
+              max-int
+    $.less-than 0.00001
+
+[] > sqrt-check-infinity-1
+  assert-that > @
+    sqrt.
+      number positive-infinity
+    $.equal-to positive-infinity
+
+[] > sqrt-check-infinity-2
```

Comment:
@levBagryansky maybe it would be better to name `sqrt-check-negative-infty`?

Reply:
@includealex I do not think so

Reply:
@levBagryansky please check this one https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html